### DISCUSSION TITLE
Alias for Credit/Debt

### DISCUSSION THREAD

**Discussion by KiTCoder** [simple-login/app#1278] [Feature Requests]
Create an alternative to Privacy.com - easily generate a number of virtual debt/credit cards with limits/monthly limits/total limits/one-time use. I feel this would be an amazing addition for security that suite well with secured e-mail aliases!

**Comment by nguyenkims** [comment]
This is indeed part of our long term roadmap, please stay tuned!

**Reply by lassomagic** [reply]
This is fantastic to hear. Just throwing in my two cents - it would be great if you could set it up so that you can make payment via credit card/debit card to create a new masked card. Privacy.com forces you to link a bank account to create a virtual/masked credit card, but many are uncomfortable with doing so. Let me know if you have any thoughts on the above! @nguyenkims

**Reply by nguyenkims** [reply]
We don't have the details on the final implementation yet but we'll try our best to make sure the process is as seamless as possible and require as little information as possible.

**Reply by harshd3shpande** [reply]
>it would be great if you could set it up so that you can make payment via credit card/debit card to create a new masked card.

That would force them to bear the risk in case of a chargeback. The implementation would most likely be tokenizing your card directly like how Google/Apple Pay do it.

**Comment by Coder84619** [comment]
Yes, please do NOT require linking to a bank account. This can cause privacy concerns as well as offering no credit card protections like fraud, purchase protection, extended warranties, etc. Please allow credit and debit card options.

**Comment by five2seven** [comment]
Following!! I would LOVE to cancel my Privacy account.

**Comment by onkq** [comment]
An update would be appreciated.

**Comment by Asherathe** [comment]
I've been eagerly awaiting this feature for some time now.

**Comment by KiTCoder** [comment]
*** 👍🏼

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Update: https://github.com/simple-login/app/discussions/482#discussioncomment-7385235

**Comment by strawrhu** [comment]
This would be tremendously helpful!

**Reply by ezbncs** [reply]
it would be nice if they did, but my credit card and bank card can do that already. so not something i would pay for since it's free through those cards. I know paypal use to be able to also do it back in the mid or so 2005 timeframe so im not sure if paypal still does it today though. i would also like to shift away from proton myself I liked them much more in there early days.
